src/cWrapper.cpp

```diff
@@ -34,8 +34,24 @@ int wsi2dcmJson(const char *inputFile, const char *outputFileMask,
                 const char *seriesId, const char *jsonFile, int retileLevels,
                 double *downsamples, bool tiled, int batchLimit, int threads,
                 bool debug) {
-  return wsiToDicomConverter::WsiToDcm::wsi2dcm(
-      inputFile, outputFileMask, frameSizeX, frameSizeY, compression, quality,
-      startOnLevel, stopOnLevel, imageName, studyId, seriesId, jsonFile,
-      retileLevels, downsamples, tiled, batchLimit, threads, debug);
+  WsiRequest request;
+  request.inputFile = inputFile;
```

Comment:
What about WsiRequest request = {
  .inputFile = inputFile,
  .outputFileMask = outputFileMask,
  ...
}
?

Reply:
well, this is C99 feature, id prefer avoid using of it

Reply:
Ah yes good point, sounds good.